cli/azd/cmd/env.go

```diff
@@ -79,6 +79,12 @@ func envActions(root *actions.ActionDescriptor) *actions.ActionDescriptor {
 		DefaultFormat:  output.EnvVarsFormat,
 	})
 
+	group.Add("get-value", &actions.ActionDescriptorOptions{
```

Comment:
I just want to start a discussion to make sure we are choosing the right command name here.  In the future I could see us also add a `config` command group under `azd env` to start manipulating config data associated with an environment in addition to `.env` key/values.  So that might look like `azd env config get <name>`

With that `azd env get-value <name>` is good, but does it keep the command plane open enough for future extensions?

Reply:
I'd prefer to not think about that for this simple change.  I don't see this add as blocking future config command.

Reply:
Agree it doesn't block, just wanted to mention it.